instrumentation/opentelemetry-instrumentation-mysqlclient/tests/test_mysqlclient_integration.py

```diff
@@ -96,6 +96,264 @@ def test_instrument_connection(self, mock_connect):
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
 
+    @mock.patch("opentelemetry.instrumentation.dbapi.instrument_connection")
+    @mock.patch("MySQLdb.connect")
+    # pylint: disable=unused-argument
+    def test_instrument_connection_enable_commenter_dbapi_kwargs(
+        self,
+        mock_connect,
+        mock_instrument_connection,
+    ):
+        cnx = MySQLdb.connect(database="test")
+        cnx = MySQLClientInstrumentor().instrument_connection(
+            cnx,
+            enable_commenter=True,
+            commenter_options={"foo": True},
+        )
+        cursor = cnx.cursor()
+        cursor.execute("Select 1;")
+        kwargs = mock_instrument_connection.call_args[1]
+        self.assertEqual(kwargs["enable_commenter"], True)
+        self.assertEqual(kwargs["commenter_options"], {"foo": True})
+
+    def test_instrument_connection_with_dbapi_sqlcomment_enabled(self):
```

Comment:
I don't know either. 🙁 MySQL drivers can't query sqlite3 which would have been handy for the from-file part. There is a 3rd party package I saw called [testing.mysqld](https://pypi.org/project/testing.mysqld/) but we're not currently using that dep and it's been a while since its last release so I'd hesitate in adding it.

Reply:
We could look into docker tests for it in the future

Reply:
@tammy-baylis-swi, maybe create an issue to add the docker-test in the future?

Reply:
Yes, new issue! https://github.com/open-telemetry/opentelemetry-python-contrib/issues/3032

Feel free to edit the issue title/description if I missed anything.

Reply:
> We could look into docker tests for it in the future

@aabmass Should I add `TODO` lines linking the [new issue](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/3032) or not necessary?

Reply:
I think it's ok to skip